lib/ansible/modules/utilities/logic/debug.py

```diff
@@ -72,4 +73,10 @@
     msg:
       - "Provisioning based on YOUR_KEY which is: '{{ lookup('env', 'YOUR_KEY') }}"
       - "These servers were built using the password of '{{ password_used }}'. Please retain this for later use."
+
+# Example that prints value with colon followed by space
```

Comment:
The example just above this one is already demonstrating the use of quotes when colon is followed by a space.

To be honest, this is not related to the **debug** module at all, but basic YAML. So adding warnings and examples to every module seems overly ridiculous. I am not at all in favor of "fixing" this here, like this.

Reply:
PS This is also the main reason why we recommend to *not* use quotes in examples for strings, unless they are absolutely required. So that users understand when they are required (rather than assume they are needed in more cases then they should).